config/jobs/kubernetes-security/generated-security-jobs.yaml

```diff
@@ -1662,7 +1662,7 @@ presubmits:
         - args:
           - --ssh=/etc/ssh-security/ssh-security
           - --repo=github.com/kubernetes-security/$(REPO_NAME)=$(PULL_REFS)
-          - --repo=github.com/kubernetes-security/kubernetes-anywhere=kubeadm-e2e
+          - --repo=k8s.io/kubernetes-anywhere=kubeadm-e2e
```

Comment:
these are the new generated paths
is this correct?

`github.com/kubernetes-security/kubernetes-anywhere` is wrong to my understanding.

Reply:
yes this is correct for now, k-s is mostly about k-s/k 👍